test/tests/WriteTest.cpp

```diff
@@ -176,4 +177,17 @@ struct WriteTest : tpunit::TestFixture {
         tester->executeWaitVerifyContent(query2);
     }
 
+    void keywordsAsValue() {
+        SData query("query: INSERT INTO stuff VALUES ( NULL, 11, 'Please update the test' );");
+        tester->executeWaitVerifyContent(query);
+
+        SData query2("query: INSERT INTO stuff VALUES ( NULL, 12, 'Do not delete the test' );");
+        tester->executeWaitVerifyContent(query2);
+
+        // Monil B - 05/26/21
+        // Uncomment this test once https://github.com/Expensify/Expensify/issues/165207 is solved
+        // SData query3("query: UPDATE stuff SET info = 'This is not a where clause';");
+        // tester->executeWaitVerifyContent(query3, "502 Query aborted");
```

Comment:
Maybe uncomment this test and make it pass with the (unexpected) current response, and leave this commented line explaining why it should be that way instead?

Reply:
Yes, can be done. It will also make more sense.